lib/gitlab/configuration.rb

```diff
@@ -32,6 +32,7 @@ def options
     def reset
       self.endpoint       = ENV['GITLAB_API_ENDPOINT']
       self.private_token  = ENV['GITLAB_API_PRIVATE_TOKEN']
+      self.auth_token     = ENV['GITLAB_API_AUTH_TOKEN']
```

Comment:
I was going with `auth_token` and `private_token` because it follows the gitlab wording, `private_token` for the personal tokens and `auth_token` for OAuth applications. You would rather use the `private_token` for both authentication schemes and have an `oauth` option to toggle between them?